src/engine/pieces/pawn.js

```diff
@@ -9,6 +9,26 @@ export default class Pawn extends Piece {
         this.name = PieceName.PAWN
     }
 
+    takeDiagonalPiece(board, color){
+        let currentSquare = board.findPiece(this);
+        let row = currentSquare.row;
+        let col = currentSquare.col;
```

Comment:
There is a nice way to define these quickly. We can use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). eg
```
let { row, col} = board.findPiece(this)
```